infrastructure/terraform/locals.tf

```diff
@@ -23,6 +23,7 @@ locals {
   function_service_plan_name = "${local.prefix}-${var.function_name_suffix}"
   cleaned_function_storage_account_name= replace("${local.function_name}}", "/[^a-z0-9]/", "")
   function_storage_account_name = lower(substr(local.cleaned_function_storage_account_name, 0, min(length(local.cleaned_function_storage_account_name), 22)))
+  azure_function_name_helloworld = "${local.function_name}-${var.azure_function_helloworld_service_name_suffix}"
```

Comment:
Can we get rid of the sample hello world function?

Reply:
we dont have a consistent health check of the service so this helps for the time being to test the know good function to test infra deployment.